sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/directconnectivity/GatewayAddressCacheTest.java

```diff
@@ -656,7 +669,8 @@ public void tryGetAddresses_SuboptimalMasterPartition_NotStaleEnough_NoRefresh()
                                                                 null,
                                                                 clientWrapper.getSpyHttpClient(),
                                                                 refreshPeriodInSeconds,
-                                                                false);
+                                                                false,
+                                                                ApiType.TABLE);
```

Comment:
should we use SQL the apiType? The direct test group will target prod sql account

Reply:
changed